src/utils/translate-app-config.ts

```diff
@@ -112,8 +112,14 @@ export function translateAppConfig(app: App): AppConfig {
         }
       }
 
+      const aliasRecordTtlInSeconds =
+        app.customDomain?.aliasRecordTtlInSeconds ?? 60;
```

Comment:
Do we really want to set an explicit default? If yes, then we maybe should use the default that the AWS console offers when creating a "simple" A record, which is 300 seconds.

![Screenshot 2021-04-16 at 13 06 56](https://user-images.githubusercontent.com/761683/115016393-417e5680-9eb5-11eb-840c-06339b0d7167.png)

Reply:
Sorry, only now saw your comments on this. In this case I would vote for 300.

Reply:
As discussed externally, no explicit default is set.